ams/server/src/main/java/com/netease/arctic/server/table/TableManager.java

```diff
@@ -1,10 +1,11 @@
 package com.netease.arctic.server.table;
 
 import com.netease.arctic.AmoroTable;
+import com.netease.arctic.ams.api.TableIdentifier;
 
 public interface TableManager extends TableRuntimeHandler {
 
-  AmoroTable<?> loadTable(ServerTableIdentifier tableIdentifier);
+  AmoroTable<?> loadTable(TableIdentifier tableIdentifier);
```

Comment:
I suggest keeping `ServerTableIdentifier` as the parameter to maintain that this interface is only for managed tables.

Reply:
fixed.